### DISCUSSION TITLE
The demo site is reallly sh...ty

### DISCUSSION THREAD

**Discussion by LaurentiuVlasa** [primefaces/community#1188] [PrimeNG] [answered]
It is very difficult to use the demo site when studying the components. It freezes my Chrome, impossible to navigate, it'svery slow. You should to something about it. Nevertheless, I like the primeng library ...
Thx

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

We're aware of the issue of late server response time, it happened with v17 upgrade, Vercel does not support angular well and we're seeking alternatives. Thanks for the feedback.
